test/tar_interpreter_test.go

```diff
@@ -0,0 +1,156 @@
+package test
+
+import (
+	"archive/tar"
+	"bytes"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/wal-g/wal-g/internal"
+)
+
+type testInterpretOptions struct {
```

Comment:
What is the reason for creating this struct? it doesn't simplify the code for reading and hide dependencies.
Use KISS principle here.

Reply:
Passing keyword arguments is redundant here.